.github/infrastructure/terraform/conformance/pubsub/aws/snssqs/snssqs.tf

```diff
@@ -0,0 +1,94 @@
+resource "aws_sns_topic" "testTopic" {
+  name = "testTopic"
+  tags = {
+    dapr-topic-name = "testTopic"
+  }
+}
+
+resource "aws_sns_topic" "multiTopic1" {
+  name = "multiTopic1"
+  tags = {
+    dapr-topic-name = "multiTopic1"
+  }
+}
+
+resource "aws_sns_topic" "multiTopic2" {
+  name = "multiTopic2"
+  tags = {
+    dapr-topic-name = "multiTopic2"
+  }
+}
+
+resource "aws_sqs_queue" "testQueue" {
+  name = "testQueue"
+  tags = {
+    dapr-queue-name = "testQueue"
+  }
+}
+
+resource "aws_sns_topic_subscription" "multiTopic1_testQueue" {
+  topic_arn = aws_sns_topic.multiTopic1.arn
+  protocol  = "sqs"
+  endpoint  = aws_sqs_queue.testQueue.arn
+}
+
+resource "aws_sns_topic_subscription" "multiTopic2_testQueue" {
+  topic_arn = aws_sns_topic.multiTopic2.arn
+  protocol  = "sqs"
+  endpoint  = aws_sqs_queue.testQueue.arn
+}
+
+resource "aws_sns_topic_subscription" "testTopic_testQueue" {
+  topic_arn = aws_sns_topic.testTopic.arn
+  protocol  = "sqs"
+  endpoint  = aws_sqs_queue.testQueue.arn
+}
+
+resource "aws_sqs_queue_policy" "testQueue_policy" {
+    queue_url = "${aws_sqs_queue.testQueue.id}"
+
+    policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Id": "sqspolicy",
+  "Statement": [
+    {
+      "Sid": "First",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "sqs:SendMessage",
+      "Resource": "${aws_sqs_queue.testQueue.arn}",
+      "Condition": {
+        "ArnEquals": {
+          "aws:SourceArn": "${aws_sns_topic.testTopic.arn}"
+        }
+      }
+    }, 
+    {
+      "Sid": "First",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "sqs:SendMessage",
+      "Resource": "${aws_sqs_queue.testQueue.arn}",
+      "Condition": {
+        "ArnEquals": {
+          "aws:SourceArn": "${aws_sns_topic.multiTopic1.arn}"
+        }
+      }
+    },
+    {
+      "Sid": "First",
+      "Effect": "Allow",
+      "Principal": "*",
```

Comment:
> Not to be a naming purist but why `CRON_COMPONENT` membership? Seems a bit odd that to run things in a different cloud you need permissions to run things in the Cron component.

CRON_COMPONENT here means the tests run on a schedule instead of on every pr. For conformance tests that will use terraform to provision resources this is currently necessary. The conformance framework doesn't support dynamic component names so we cant avoid resource conflicts if multiple of these jobs run. Certification tests this isn't an issue.